.circleci/config.yml

```diff
@@ -30,16 +30,21 @@ jobs:
             fi
   build_and_test_and_deploy_ui:
     docker:
-      - image: allofustest/workbench:buildimage-0.0.4
+      - image: allofustest/workbench:buildimage-0.0.5
     working_directory: ~/workbench
     environment: []
     steps:
       - checkout
       - run:
-          working_directory: ~/
-          name: Set up swagger CLI
+          working_directory: ~/workbench/ui
+          name: Fetch submodules
+          command: |
+            git submodule update --init libproject/utils
+      - run:
+          working_directory: ~/workbench/ui
+          name: Download Swagger CLI
           command: |
-            cd workbench/tools && ./download_swagger_cli.sh
+            ruby -r ./libproject/devstart.rb -e download_swagger_codegen_cli
```

Comment:
Should `devstart.rb` really be named something more general? It seems like it's taking on other responsibilities. Or should we split out other actions into separate scripts?